json-guides/microprofile-config.json

```diff
@@ -1138,7 +1138,7 @@
         {
             "name": "WhatNext",
             "title": "Great work!  You're done!",
-            "description": ["You learned how to use MicroProfile Config to separate configuration from code by using injection, configuring with properties files, configuring with environment variables, and configuring with system properties. You also learned the default ordinals for each of these configuration sources and how to override them."
+            "description": ["You learned how to use MicroProfile Config to separate configuration from code by using injection, configuring with properties files, environment variables, and system properties, and by providing values within the Liberty server.xml file. You also learned the default ordinals for each of these configuration sources and how to override them."
```

Comment:
![image](https://user-images.githubusercontent.com/30509453/78033357-e7ebce00-732b-11ea-9791-f68477291434.png)
environment variables, system properties, and by providing values within the Liberty server.xml

Reply:
Rewrote the paragraph.   See update